src/Store/auth-context.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+
+const AuthContext = React.createContext({
+    token: '',
+    isLoggedIn: false,
+    login: (token) => { },
+    logout: () => [],
+    isAdmin: false,
+});
+
+export const AuthContextProvider = (props) => {
+    const initialToken = localStorage.getItem('token');
+    const [token, setToken] = useState(initialToken);
+
+    const [admin,setAdmin] = useState(false);
+
+    const userIsLoggedIn = !!token;
+
+    const loginHandler = (props) => {
+        setToken(props.token);
+        localStorage.setItem('token',token);
+
+        if(props.isAdmin==="admin") {
+            setAdmin(true);
+        }
+        else {
+            setAdmin(false);
+        }
+        localStorage.setItem('isAdmin', admin);
+    };
```

Comment:
OPTIONEEL - Dit zou eventueel in een Guard Clause kunnen (eerst de error / false opvangen en vervolgens verder met het uitvoeren van de rest van de code):

if(props.isAdmin!=="admin"){
   setAdmin(false)
}
setAdmin(true);
localStorage.setItem('isAdmin', admin)

Reply:
Ah thanks! Hier liep ik steeds tegenaan waarom ik het niet voor elkaar kreeg de else weg te laten zoals jullie dat doen. 
Neem ik later mee!